src/sonic-yang-models/yang-models/sonic-qos-pfc.yang

```diff
@@ -0,0 +1,82 @@
+module sonic-qos-pfc {
+    namespace "http://github.com/Azure/sonic-qos-pfc";
+    prefix sonic-qos-pfc;
+
+    yang-version 1.1;
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC QoS Priority-based Flow Control";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-qos-pfc {
+
+        container PFC_WD {
+            list PFC_WD_LIST {
+                key "ifname";
+
+                leaf ifname {
+                    mandatory true;
```

Comment:
please remove mandatory=true for key-attribute (ifname). it's not required for key-attributes, since it's mandatory implicitly. Generally, we add mandatory=true for leaf-attributes, if backend always expect that attribute on entry creation.